test-requirements.txt

```diff
@@ -0,0 +1 @@
+odoo14-addon-partner-tz @ git+https://github.com/trobz/partner-contact@14.0-mig-partner_tz#subdirectory=setup/partner_tz
```

Comment:
Has been merged, we can remove it now
- https://github.com/OCA/partner-contact/pull/1043

Reply:
Hi @sebalix , sorry for late reply, I pushed the update